src/renderer/components/SettingsModal/index.jsx

```diff
@@ -51,6 +51,7 @@ export default class SettingsModal extends React.Component {
     this.setState({
       nWorkersOptions: nWorkersOptions,
     });
+    this.isDevMode = ipcRenderer.sendSync(ipcMainChannels.IS_DEV_MODE);
```

Comment:
Whoops, I think one of my comments on the last round may have gotten lost in the sea of comments. Can we change this to an `await invoke / handle` pair because it's strongly discouraged to make blocking sync calls in the renderer process, and because `handle` seems expressly designed to `return` a value?

Reply:
Ooops, sorry I missed that! It's fixed now